.shippable.yml

```diff
@@ -13,5 +13,5 @@ build:
     - pip install -r tests_requirements.txt
     - pip install -e .
     - mkdir -p shippable/{testresults,codecoverage}
-    - pytest --junitxml=shippable/testresults/nosetests.xml tests
-    - pytest --cov=west --cov-report=xml:shippable/codecoverage/coverage.xml tests
+    - PYTHONPATH=src/west pytest --junitxml=shippable/testresults/nosetests.xml tests
```

Comment:
Got any suggestions for how to fix it? Didn't know we had a thing for running tests through `setup.py`.

Reply:
I guess the setup.cfg alias entry will need some changes? I'm not sure exactly why this patch set breaks it, is it the elimination of relative imports?

Reply:
> I actually think it'd be nice to try to have per-installation configuration to start with -- so `TOP/west/westconfig.yml` -- thoughts?

I'd prefer some simple `.ini`-like format, like Git uses. Think `.yml` might be overkilling it.

Reply:
This seems nice: https://docs.python.org/3/library/configparser.html

Reply:
I'm not sure why it worked before to be honest. I don't think `pytest` can guess where the code-to-be-tested is, and in this case it's two directories down as well.

Google suggests setting `PYTHONPATH`. Did you put in some helper for `pytest` to localize things somewhere?

Reply:
(`PYTHONPATH=src/west python3 setup.py test` works by the way.)

Reply:
Looking at https://github.com/pytest-dev/pytest/issues/2421, it seems that `PYTHONPATH` might be needed (they recommend testing against installed packages too).

Still no idea how it worked before. Maybe it just happened to work through the test directories being laid out the same way and some magic relative import thing happening, or maybe I'm still missing something...

Reply:
I updated the instructions for running the tests in the README.

I think `pytest-runner` assumes that `setup.py` will install everything that needs to be tested (so that it ends up in `sys.path`). That causes issues in our case, since only the bootstrap script gets installed.

Reply:
Finally figured out what's going on:

When the bootstrap script is installed with `pip install -e` (as opposed to a plain `pip install`), the directory `west/src` ends up in `sys.path`, since the `bootstrap` package happens to be in there.

That indirectly makes `west/src/west` importable, provided it is a module, which is how the tests found them.

IOW, `pytest` previously relied on having run `pip install -e .` first.